wgsl/index.bs

```diff
@@ -11575,6 +11592,31 @@ See [[#function-calls]].
         as `select(f[i], t[i], cond[i])`.
 </table>
 
+### `wgsl_version_at_least` ### {#wgsl-version-at-least-builtin}
+<table class='data builtin'>
+  <tr algorithm="wgsl_version_at_least">
+    <td style="width:10%">Overload
+    <td class="nowrap"><xmp highlight=rust>
+@const fn wgsl_version_at_least(major: T,
+                                minor: T) -> bool
+</xmp>
+  <tr>
+    <td style="width:10%">Parameterization
+    <td>`T` is AbstractInt, i32, or u32
+  <tr>
+    <td>Description
+    <td>Where *WGSLMajor* is the WGSL [=major version number=], and *WGSLMinor* is the [=minor version number=],
+        returns:
+
+        ((`major` &lt; *WGSLMajor*) | ((`major` == *WSGSLMajor*) & (`minor` &le; *WGSLMinor*)))
+</table>
+
+<div class='example wgsl global-scope' heading="Ensure a shader only runs on an implementation with WGSL version 1.0">
```

Comment:
This description seems wrong. This shader will run on an implementation that supports 1.0 or greater.